spec/rspec/mocks/null_object_double_spec.rb

```diff
@@ -76,7 +76,7 @@ module Mocks
         expect(val).to equal(@double)
       end
 
-      it 'allows unexpected message sends using `send`' do
+      it 'allows unexpected message sends using `__send__`' do
```

Comment:
Can you leave this as it was? `__send__` is the same as `send` but we use the double underscore version "just to be sure" TM

Reply:
I find it confusing having the same description, but :ok:, I'll revert

Reply:
Oh sorry I take it back, please keep!